base/rational.jl

```diff
@@ -95,6 +95,22 @@ end
 function write(s::IO, z::Rational)
     write(s,numerator(z),denominator(z))
 end
+function parse(::Type{Rational{T}}, s::AbstractString) where T<:Integer  
+    ss = split(s, '/'; limit = 2)
+    if isone(length(ss))
+        return Rational{T}(parse(T, ss[1]))
```

Comment:
```suggestion
        return Rational{T}(parse(T, s))
```
I should have suggested that from the get-go, it's simpler.